src/shared/components/Select/component.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { SelectedWrapper, SelectWrapper, SelectItemsList, SelectItemsListItem } from './styles';
+import { DropdownIcon } from '../../icons';
+
+export type SelectProps = {
```

Comment:
let's put these type into `shared/types`; and import from there.

Reply:
done